aspen/simplates/renderers/__init__.py

```diff
@@ -81,6 +81,49 @@ class CheeseFactory(Factory):
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import sys
+import pkg_resources
+
+from ... import log_dammit
+
+# Built-in renderers
+BUILTIN_RENDERERS = [ 'stdlib_format'
+                    , 'stdlib_percent'
+                    , 'stdlib_template'
+                    , 'json_dump'
+                    , 'jsonp_dump'
+                     ]
+
+RENDERERS = BUILTIN_RENDERERS[:]
+
+for entrypoint in pkg_resources.iter_entry_points(group='aspen.renderers'):
```

Comment:
yes, though once we split it into a different module, it will be `simplate.renderers` I suppose.